collectors/slurm/src/database.rs

```diff
@@ -90,7 +90,23 @@ impl Database {
             .await?
         {
             Some(Row { lastcheck, jobid }) => {
-                Ok((Local.from_local_datetime(&lastcheck).unwrap(), jobid))
+                //Ok((Local.from_local_datetime(&lastcheck).unwrap(), jobid))
+
+                match Local.from_local_datetime(&lastcheck) {
+                    LocalResult::Single(datetime) => Ok((datetime, jobid)),
+                    LocalResult::Ambiguous(datetime1, datetime2) => {
+                        tracing::info!("Ambiguous local time, ranging from t1 = {:?} to t2 = {:?} . Choosing t1 as timestamp", datetime1, datetime2);
+                        Ok((datetime1, jobid))
+                    }
+                    _ => {
```

Comment:
Instead of using `_` I think it would be better to explicitly match the third case (`chrono::LocalResult::None`), in case there would be additional variants of the enum in the future.